### DISCUSSION TITLE
I’m getting a bit lost on where  marked symbols for lip synch are?

### DISCUSSION THREAD

**Discussion by MojoMan49** [keiwando/toonsquid#173] [Show and tell]
![D90C48D0-8DA0-440A-9088-0CCA71F66D20](https://user-images.githubusercontent.com/127110520/223274724-d21d8126-4724-4514-ae6c-5116389c5ef3.jpeg)

**Comment by MojoMan49** [comment]
I found a “how to” on Youtube on how to do symbols with markers that can be used to lip synch and I’ll be darned if I’m not more confused now than ever. I found what I thought was a really promising Youtube, but there is clearly something my 3.7.1 versions is just different.

**Reply by Cat-Butler** [reply]
You are most of the way there. 

(There are at least 2 ToonSquid lipsync videos now on YouTube, one 4-minute one by ToonSquid [keiwan], which is good, and a 2-minute one by Free at sea, which I haven't watched yet).

You might also want to read the manual page, which explains it well 
https://toonsquid.com/handbook/symbols/marker_animation/

**Optional but very useful:**
When you edit the Eye Droop symbol layer [or the All Mouth layer, if appropriate] (tap Library, tap and hold on the symbol thumbnail, Edit will appear along the bottom of the Library panel), does each frame have it's own Frame Marker "name", and does each drawing only take up a single frame? This is not absolutely essential, but will definitely make it easier.
Next, change back to the main timeline. 

**You've already done this:**
Select your symbol layer (as in your screenshot).
Tap the Inspector icon (as in your screenshot).

**The essential next step you might have missed:**
You can scroll/drag vertically in the Inspector/Properties panel to reveal the bottom of your Inspector/Properties panel for the selected symbol layer. 
Hiding down underneath the "Muted" switch should be a Time slider, which controls the frame displayed from your selected symbol layer.
If you've marked (named/labelled) each drawing in your symbol layer, and each drawing is only a single frame, you will also have an option "Select from Markers".

**(Good) Using the slider:**
Choose a frame on the main timeline. Drag the Time slider to choose a frame in the symbol layer (for example, if you had eyes closed on frame 3 of your symbol, move the slider to 3; similarly, if you had the mouth position for O on frame 5 of your All Mouth symbol layer, with the symbol layer selected in the timeline, move the Time slider to 5). If your symbol uses single-frame labelled drawings, your keyframe will "hold" until the next keyframe.

**(Better) Using the "Select from Markers" option:** 
If you have labelled (named) each single-frame drawing in your symbol layer, it gets even better: tap on Select from Markers in the Inspector panel (will appear automatically if your symbol has all single framed labelled markers), and you get a grid of thumbnails for each eye or mouth position/shape. 
Simply change to the appropriate frame in the main timeline, and tap the correct thumbnail to add a Time keyframe with the correct drawing from the symbol layer, set to "hold" mode.

**Reply by MojoMan49** [reply]
Also wanted to let you know that using frame markers in symbols is a great way to choose “eyes” and put mouth into sync - Thanks again

**Comment by MojoMan49** [comment]
@Cat-Butler - I really want to thank you for the insights and encouragement.

So, still a long way to go. I’m just now beginning to get the symbol/marker thing going.

https://user-images.githubusercontent.com/127110520/224200837-29ae43ab-1896-4b76-b576-edc5a46cfded.MOV

I have no idea how “Talk Like  a Pirate Day” was the gyst of this mp4

**Reply by Cat-Butler** [reply]
Hey @MojoMan49, just wanted to say thanks for the kind words (sorry about the late reply, I've had a hectic couple of weeks) and I'm glad to read that you're getting the frame/marker approach under control.
With ToonSquid being such a new app, we're all still learning.

**Reply by MojoMan49** [reply]
Hey, this is a volunteer thing and I appreciate you kind words.

https://user-images.githubusercontent.com/127110520/227006177-91574b93-ff21-49ac-8107-5f17f34e0bbd.MOV

I’ll be begging the developer to make “symbols” in the app layer universal to ALL projects in the application.

Having to create mouth symbols, whether for the same character or as the basis for another one is a genuine production problem… Your thoughts?

**Comment by MojoMan49** [comment]
https://user-images.githubusercontent.com/127110520/227607292-c968bca1-63dd-4060-8279-a22b01d109d4.mov

Added another character “Jonah” and used folders to keep all his component symbols in a handy place.

I almost forgot, the import multiple images is a real great improvement!
